time times
---
float64 x
float64 dx
float64 ddx
float64 y
float64 dy
float64 ddy
float64 z
float64 dz
float64 ddz
bool flag_begin